Uncaught (in promise) Error: Cannot find module '../node_modules/string_decoder/index.js'

### Prerequisites

* [X] Put an X between the brackets on this line if you have done all of the following:
    * Reproduced the problem in Safe Mode: https://flight-manual.atom.io/hacking-atom/sections/debugging/#using-safe-mode
    * Followed all applicable steps in the debugging guide: https://flight-manual.atom.io/hacking-atom/sections/debugging/
    * Checked the FAQs on the message board for common solutions: https://discuss.atom.io/c/faq
    * Checked that your issue isn't already filed: https://github.com/issues?utf8=✓&q=is%3Aissue+user%3Aatom
    * Checked that there is not already an Atom package that provides the described functionality: https://atom.io/packages

### Description

When the `github` package starts up its filesystem watcher the following stack trace is thrown to the console:

```
Uncaught (in promise) Error: Cannot find module '../node_modules/string_decoder/index.js'
    at Module._resolveFilename (module.js:470:15)
    at Module._resolveFilename (C:\temp\atom\553dec3e7\Atom x64\resources\electron.asar\common\reset-search-paths.js:35:12)
    at Function.get_Module._resolveFilename (C:\temp\atom\553dec3e7\Atom x64\resources\app\src\module-cache.js:354:58)
    at Module.require (file:///C:/temp/atom/553dec3e7/Atom%20x64/resources/app.asar/static/index.js:40:43)
    at require (internal/module.js:20:19)
    at customRequire (C:\temp\atom\553dec3e7\Atom x64\resources\app\static\<embedded>:96:26)
    at get_Decoder (C:\temp\atom\553dec3e7\Atom x64\resources\app\node_modules\split\index.js:12:37)
    at split (C:\temp\atom\553dec3e7\Atom x64\resources\app\node_modules\split\index.js:20:28)
    at C:\temp\atom\553dec3e7\Atom x64\resources\app\node_modules\what-the-status\index.js:37:18
    at Promise (<anonymous>)
    at parse (C:\temp\atom\553dec3e7\Atom x64\resources\app\node_modules\what-the-status\index.js:16:16)
    at C:\temp\atom\553dec3e7\Atom x64\resources\app\node_modules\github\lib\git-shell-out-strategy.js:567:60
    at Generator.next (<anonymous>)
    at step (C:\temp\atom\553dec3e7\Atom x64\resources\app\node_modules\github\lib\git-shell-out-strategy.js:92:197)
    at C:\temp\atom\553dec3e7\Atom x64\resources\app\node_modules\github\lib\git-shell-out-strategy.js:92:367
    at <anonymous>
```

### Steps to Reproduce

1. Enable "Filesystem Event Diagnostics" in the `github` package
2. Open a project folder that contains a git repository
3. Open the console and see the errors when the `fs watcher` starts up

**Expected behavior:** [What you expect to happen]

No errors.

**Actual behavior:** [What actually happens]

Errors.

**Reproduces how often:** [What percentage of the time does it reproduce?]

100% of the time.

Note that if the `github` package's `fs watcher` doesn't initialize, this doesn't show normally.

### Versions

This started happening on builds after https://github.com/atom/atom/commit/553dec3e7a84ccb1965f557a1aad36822025905e.

Seen on both Windows (10 x64 Enterprise) and macOS builds.

### Additional Information

The changes to `text-buffer` brought in by that commit can be found here: https://github.com/atom/text-buffer/compare/v13.12.4...v13.13.0. The suspicious commit being merging https://github.com/atom/text-buffer/pull/280 as it introduced `cpu-cli` as a direct `dependency` instead of a `devDependency` as it should have been. That package has `string_decoder` as a dependency, and although `string_decoder` was already in the tree that PR moved it from a `devDependency` to a regular `dependency`.

I _believe_ that https://github.com/atom/text-buffer/pull/296 will fix it, but have been unable to test without a version of `text-buffer` on NPM that includes it.

--- a/package.json
+++ b/package.json
@@ -92,7 +92,7 @@
     "solarized-dark-syntax": "1.1.5",
     "solarized-light-syntax": "1.1.5",
     "about": "1.9.1",
-    "archive-view": "0.64.6",
+    "archive-view": "0.65.0",
     "autocomplete-atom-api": "0.10.7",
     "autocomplete-css": "0.17.5",
     "autocomplete-html": "0.8.4",

--- a/script/lib/generate-startup-snapshot.js
+++ b/script/lib/generate-startup-snapshot.js
@@ -42,7 +42,6 @@ module.exports = function (packagedAppPath) {
         requiredModuleRelativePath === path.join('..', 'node_modules', 'atom-keymap', 'lib', 'command-event.js') ||
         requiredModuleRelativePath === path.join('..', 'node_modules', 'babel-core', 'index.js') ||
         requiredModuleRelativePath === path.join('..', 'node_modules', 'cached-run-in-this-context', 'lib', 'main.js') ||
-        requiredModuleRelativePath === path.join('..', 'node_modules', 'decompress-zip', 'lib', 'decompress-zip.js') ||
         requiredModuleRelativePath === path.join('..', 'node_modules', 'debug', 'node.js') ||
         requiredModuleRelativePath === path.join('..', 'node_modules', 'git-utils', 'src', 'git.js') ||
         requiredModuleRelativePath === path.join('..', 'node_modules', 'glob', 'glob.js') ||

--- a/script/lib/generate-startup-snapshot.js
+++ b/script/lib/generate-startup-snapshot.js
@@ -63,6 +63,7 @@ module.exports = function (packagedAppPath) {
         requiredModuleRelativePath === path.join('..', 'node_modules', 'temp', 'lib', 'temp.js') ||
         requiredModuleRelativePath === path.join('..', 'node_modules', 'tmp', 'lib', 'tmp.js') ||
         requiredModuleRelativePath === path.join('..', 'node_modules', 'tree-sitter', 'index.js') ||
+        requiredModuleRelativePath === path.join('..', 'node_modules', 'yauzl', 'index.js') ||
         requiredModuleRelativePath === path.join('..', 'node_modules', 'winreg', 'lib', 'registry.js')
       )
     }

